### PR TITLE
Clone collection edtior

### PR DESCRIPTION
**Pull request by tbezman**
### Incremental PR
This PR is a starting point. I did a ton of work on the new collection editor. I came back and wanted to get this in so the future diffs are clearer. 

This PR does two things
1. Clones the old collection editor to a new directory
2. Implements the new sidebars (I didn't want to include this but I would have had to untangle some more commits)

image

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -39,10 +39,25 @@ module.exports = {
...
+      rules: {
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: ['~/components/ManageGallery/OrganizeCollection/*'],
+          },
+        ],
+      },
+    },
+
```
Comment (tbezman) [inline .eslintrc.cjs:60]: really dig this for the migration period

File: pages/gallery/[galleryId]/edit.tsx [hunk 2]
```diff
@@ -1,19 +1,78 @@
+import { GetServerSideProps } from 'next';
```
Comment (tbezman) [inline pages/gallery/[galleryId]/edit.tsx:1]: feature flagging the new editor. right now its disable for all groups

File: src/utils/graphql/isFeatureEnabled.tsx [hunk 3]
```diff
@@ -8,26 +8,31 @@ import { removeNullValues } from '~/utils/removeNullValues';
 export enum FeatureFlag {
```
Comment (tbezman) [inline src/utils/graphql/isFeatureEnabled.tsx:8]: feature flags

File: src/components/GalleryEditor/PiecesSidebar/PiecesSidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,274 @@
+import keyBy from 'lodash.keyby';
```
Comment (tbezman) [inline src/components/GalleryEditor/PiecesSidebar/PiecesSidebar.tsx:1]: new right hand sidebar

File: src/components/GalleryEditor/CollectionSidebar/CollectionSidebar.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import { graphql, useFragment } from 'react-relay';
```
Comment (tbezman) [inline src/components/GalleryEditor/CollectionSidebar/CollectionSidebar.tsx:1]: new left hand sidebar

File: .eslintrc.cjs (hunk 1 above)
Comment (Robinnnnn) [reply .eslintrc.cjs:60]: this is awesome

**Review by Robinnnnn — APPROVED** [review]

**Pull request merged**